Tests/SKSwiftPMWorkspaceTests/SwiftPMWorkspaceTests.swift

```diff
@@ -267,9 +267,18 @@ final class SwiftPMWorkspaceTests: XCTestCase {
       let arguments = ws.settings(for: aswift.asURI, .swift)!.compilerArguments
       check(aswift.pathString, arguments: arguments)
       checkNot(bswift.pathString, arguments: arguments)
-      check(
-        "-I", packageRoot.appending(components: "Sources", "libC", "include").pathString,
-        arguments: arguments)
+      // Temporary conditional to work around revlock between SourceKit-LSP and SwiftPM
+      // as a result of fix for SR-12050.  Can be removed when that fix has been merged.
+      if arguments.joined(separator: " ").contains("-Xcc -I -Xcc") {
```

Comment:
Concatenating the command line without quoting would normally not be acceptable, but this is a temporary condition to determine whether SwiftPM has the fix or not.  This will be removed once the fix for SR-12050 has been integrated into SwiftPM.

Reply:
This has been tested by temporarily changing the dependency to be the SwiftPM PR branch containing the fix for SR-12050.